### ISSUE TITLE
Version v1.0.19 of Extension breaks previously generated tests

### ISSUE THREAD

**Issue by jambonrose** [IBM-Blockchain/blockchain-vscode-extension#1957]
## Description

When upgrading to version 1.0.19 of the extensions, functional tests previously generated by the extension cease working, because a number of paths (to the gateway JSON file and the wallet admins) have been moved.

## Expected Behavior

I would have expected:

- these changes to be documented in the Changelog
- for this stable release with breaking changes to be a major number release, not a patch release

## Possible Fix

It took us some time to figure out exactly what was going on, because some of the errors raised looked like error below.

```
Error: identity 'admin' isn't enrolled, or loaded
      at FileSystemWallet.setUserContext (node_modules/fabric-network/lib/impl/wallet/basewallet.js:56:10)
```

Once we realized v.1.0.19 had changed paths to files for the local setup, we re-generated the functional tests, and then began to adjust back to our own code. This took several hours, as we had made changes to `ts-smart-contract-util.ts` to solve for items like #1628 and to allow for gateway reuse (to avoid timeout errors).

I think the extension could benefit from testing on this front. I think developers using the extension would benefit from more verbose change-logs and more instructions above version migrations.

## Steps to Reproduce

1. Generate functional tests in a previous release
2. Upgrade the extensions
3. See errors generated by functional tests in latest release

## Related issues

- #1870
- #1687
- #1642

## Your Environment
* IBM Blockchain Platform Extension Version: v1.0.19
* Visual Studio Code version: 1.39.2 
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1): Node v10.18.1, npm 6.13.6
* Operating System and version: macOS 10.14.6 
* Link to your project: Unavailable

**Comment by cazfletch** [comment]
Hi @jambonrose, yes agreed we should have made it more clear in our change-log about this change.

**Comment by cazfletch** [comment]
We should update the readme to point out that should developers wish to persist the generated tests that they should also export the wallet and connection profile and change the file paths to them. This is because the file paths we currently use point to our internal directories so that the generated tests will work out of the box, but it isn't recommended to keep those paths as things do change.

**Comment by erin-hughes** [comment]
Closed by #2076
